integration_tests/src/main/python/hive_delimited_text_test.py

```diff
@@ -143,13 +143,9 @@ def read_impl(spark):
     # Custom datasets
     ('hive-delim-text/Acquisition_2007Q3', acq_schema, {}),
     ('hive-delim-text/Performance_2007Q3', perf_schema, {'serialization.null.format': ''}),
-    pytest.param('hive-delim-text/Performance_2007Q3', perf_schema, {},
-                 marks=pytest.mark.xfail(reason="GPU treats empty strings as nulls."
-                                                "See https://github.com/NVIDIA/spark-rapids/issues/7069.")),
+    pytest.param('hive-delim-text/Performance_2007Q3', perf_schema, {}),
```

Comment:
Nit:
```suggestion
    ('hive-delim-text/Performance_2007Q3', perf_schema, {}),
```
I'm glad this is resolved.